### ISSUE TITLE
[a11y]: Toggletip content only being read by screen readers on first activation

### ISSUE THREAD

**Issue by patrickobrien1** [carbon-design-system/carbon#19338]
### Package

@carbon/react

### Browser

Firefox

### Operating System

MacOS

### Package version

1.76.0

### React version

16.14.0

### Automated testing tool and ruleset

N/A

### Assistive technology

Jaws, Voiceover, NVDA

### Description

The toggletip implementation currently seems to prevent the expanded dialog information only upon the first activation of the toggletip button. If a user opens a toggletip, closes it, and then opens it again the toggletip content will not be read upon the second activation of the toggletip.

### WCAG 2.1 Violation

1.3.1 Info and Relationships

### Reproduction/example

https://stackblitz.com/edit/github-ingubaav?file=src%2FApp.jsx&preset=node=

### Steps to reproduce

1. Activate a screen reader
2. Provide keyboard focus to the toggletip button. Name role and label should be read
3. Activate the button using the enter key. Name role and label should be read followed by toggletip content
4. Deactivate the button using the enter key. The name and role should be read but the toggletip content is read
5. Re-activate the toggletip button using the enter key. From this point forward activating/deactivating will not cause toggletip content to be read

### Suggested Severity

None

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [x] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems
